tests/explainers/test_generic_explainer.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+
+from pyreal.explainers import Explainer
+from pyreal.transformers import FeatureSelectTransformer
+
+
+def test_init_invalid_transforms(regression_no_transforms):
+    invalid_transform = "invalid"
+    with pytest.raises(TypeError):
+        Explainer(regression_no_transforms["model"], regression_no_transforms["x"],
+                  m_transformers=invalid_transform)
+    with pytest.raises(TypeError):
+        Explainer(regression_no_transforms["model"], regression_no_transforms["x"],
+                  e_transformers=invalid_transform)
+    with pytest.raises(TypeError):
+        Explainer(regression_no_transforms["model"], regression_no_transforms["x"],
+                  i_transformers=invalid_transform)
+
+
+def test_init_invalid_model():
+    invalid_model = []
+    with pytest.raises(TypeError):
+        Explainer(invalid_model, pd.DataFrame([0]))
+
+
+def test_run_transformers(regression_one_hot):
```

Comment:
I don't think you need this test here, the functions being tested are in the BaseExplainer class and should be tested in that classes tests.

Reply:
I don't actually know what tests we need here. What I did was copying the tests in `test_base_explainer.py` but using the default explainer instead. The only new variable is the `scope`, so I reproduce these tests using different options of `scope`

Reply:
We generally only need tests that include new functionality. The `transform_to_x_XXX` functions do not use `scope`, so we don't need to test them here.